appveyor.yml

```diff
@@ -0,0 +1,77 @@
+os: Visual Studio 2013
```

Comment:
This config is the money, it creates the 64 / 32 bit Windows prebuilds on AppVeyor, running bootstrap.sh / configure is necessary for generating the config.h file, If you have time / are bored, it would be interesting to see if these configs actually differ from each other (64 vs 32 bit) or actually just save them and include them in your own "src" folder, that would allow you to get rid of all the msys2 stuff, and simply run "cmake --build" or the likes...